README.md

```diff
@@ -32,3 +37,5 @@ $ ./deploy.sh <lambda-function-name>
 ```
 
 …where `<lambda-function-name>` corresponds to the name of the Lambda function in AWS. The deploy script requires the [AWS CLI](https://aws.amazon.com/cli/) to be installed.
+
+To test the deploy script, prepend `DEPLOY=true` to the command above. This will run all build steps, but won't update the Lambda code.
```

Comment:
Should be this `DRYRUN=true`?

Reply:
Ha, yep. I had deploying on the mind, I guess. Thanks!

Reply:
Fixed